### ISSUE TITLE
syntax error with complex configuration

### ISSUE THREAD

**Issue by GeorgFleig** [vmatare/thinkfan#12]
Hi,
I have some trouble setting up thinkfan using complex mode. I am running the latest version of thinkfan on a T500 which provides 16 sensors. However when using the complex configuration file provided by this repo I get the following error message:

```
root@tp:/etc# /usr/local/sbin/thinkfan 

ERROR: /etc/thinkfan.conf:65: Syntax error:
{ level 0   # the fan level
^
```
As far as I understand I don't need to adjust the provided config file as the number of sensors is the same (16). Did I miss something?

Thanks for this great tool and the support!

**Comment by vmatare** [comment]
Thanks for the report, this is indeed a bug. Some of the features used in that example config are still missing from the config parser.

**Comment by EvansMike** [comment]
Just to say I see a slightly different message using the complex config.

ERROR: Invalid config: Inconsistent limit length

Thanks

**Comment by vmatare** [comment]
Yup, there are actually multiple issues here, two of which have been fixed by the two last commits. The third requires some larger changes in the parser which will take some time. As a workaround, you can remove the dots from the fan levels and replace them with sensible temperature limits, or you can use simple mode with correction values.

**Comment by EvansMike** [comment]
Thanks, that works. :)

**Comment by vmatare** [comment]
This is *supposed* to be fixed by commit #5346b6e, so anyone affected by this might want to retry with the current master.

**Comment by GeorgFleig** [comment]
Thanks, seems to be working for me with the current version.
